src/Uno.Toolkit.RuntimeTests/Tests/AutoLayoutTest.cs

```diff
@@ -31,6 +31,38 @@ namespace Uno.Toolkit.RuntimeTests.Tests;
 [RunsOnUIThread]
 internal class AutoLayoutTest
 {
+	[TestMethod]
+	[RequiresFullWindow]
```

Comment:
this doesnt need full window, right?
since you could just force an bigger than necessary size on `SUT`